README.md

```diff
@@ -40,3 +52,12 @@ Vim command:
 ```
 :CSScomb
 ```
+
+## Suggested Configuration
+
+```VIML
+"mnemonic: 'beatify css'
```

Comment:
It took me a while to figure out that this comment is actually meaningful.
Maybe you could write some sort of explanation for those people who don't get along with `.vimrc` files at all, so they can understand what this line does. Something like: "Map `bc` to run CSScomb. `bc` stands for `beautify css`". Or something neater :)